src/crypto/mod.rs

```diff
@@ -40,9 +41,9 @@ impl fmt::Debug for Hash {
 #[derive(Debug)]
 pub struct Prg(ChaCha12Rng);
 
-pub fn commit(key: &[u8; KEY_SIZE], value: &[u8]) -> Hash {
+pub fn commit(key: &[u8; KEY_SIZE]) -> Hash {
     let mut hasher = blake3::Hasher::new_keyed(key);
-    hasher.update(value);
+    hasher.update("REVERIE".as_bytes());
```

Comment:
@rot256  Since the hashes passed to `commit` were always empty anyways, I eliminated them and instead used a fixed string. I think we could go further and eliminate this step entirely without violating any of KKW's expectations. So, instead of using a keyed hash of a fixed string, we'd instead just using an unkeyed hash of the key. Does that sound right?

Reply:
As far as I can see it is only used in `crypto/merkle/set.rs`, which we can eliminate altogether.